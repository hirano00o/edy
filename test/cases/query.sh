#!/bin/bash

SCRIPT_ROOT_DIR=$1
TEST_NAME=$(basename "$0" | sed "s/\..*//")

# aws dynamodb query --table-name User --key-condition-expression ID=:id \
#   --expression-attribute-values "{\":id\":{\"N\":\"1\"}}" --endpoint-url http://localhost:8000
CMD="edy q -t User -p 1 --local 8000"

. "${SCRIPT_ROOT_DIR}"/helper.sh

run_such_query_helper
