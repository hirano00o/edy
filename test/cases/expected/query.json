[
  {
    "Address": {
      "City": "Little Rock",
      "State": "Arkansas"
    },
    "Age": 20,
    "Birthday": {
      "Day": 12,
      "Month": 8,
      "Year": 2000
    },
    "Email": "alice@example.com",
    "ID": 1,
    "Name": "Alice"
  }
]
